README.md

```diff
@@ -1,51 +1,158 @@
-# Introduction
+# aws-nitro-enclaves-k8s-device-plugin
 
-The Nitro Enclaves [Device Plugin](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/) gives your pods and containers the ability to access the [Nitro Enclaves device driver](https://docs.kernel.org/virt/ne_overview.html). The device plugin works with both [Amazon EKS](https://aws.amazon.com/eks/) and self-managed Kubernetes nodes.
+The Nitro Enclaves [Device Plugin](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/) gives your pods and containers the ability to access
+the [Nitro Enclaves device driver](https://docs.kernel.org/virt/ne_overview.html). The device plugin works with both [Amazon EKS](https://aws.amazon.com/eks/) and self-managed Kubernetes nodes.
 
-[AWS Nitro Enclaves](https://aws.amazon.com/ec2/nitro/nitro-enclaves/) is an [Amazon EC2](https://aws-content-sandbox.aka.amazon.com/ec2/) capability that enables customers to create isolated compute environments to further protect and securely process highly sensitive data within their EC2 instances.
+[AWS Nitro Enclaves](https://aws.amazon.com/ec2/nitro/nitro-enclaves/) is
+an [Amazon EC2](https://aws-content-sandbox.aka.amazon.com/ec2/) capability that enables customers to create isolated compute environments to further protect and securely process highly sensitive data within their EC2 instances.
+
+-------
+## Prerequisites
 
-# Prerequisites
 To utilize this device plugin, you will need:
 
-  - A configured Kubernetes cluster.
-  - At least one enclave-enabled node available in the cluster. An enclave-enabled node is an EC2 instance with the **EnclaveOptions** parameter set to **true**. For more information on creating an enclaving an enclave-enabled node, review the using [Nitro Enclaves with EKS user guide](https://docs.aws.amazon.com/enclaves/latest/user/kubernetes.html).
+- A configured Kubernetes cluster.
+- At least one enclave-enabled node available in the cluster. An enclave-enabled node is an EC2 instance with the *
+  *EnclaveOptions** parameter set to **true**. For more information on creating an enclave-enabled EKS worker node,
+  review the using [Nitro Enclaves with EKS user guide](https://docs.aws.amazon.com/enclaves/latest/user/kubernetes.html).
 
 To build the plugin, you will need:
-  - Docker
+- Docker
 
-# Usage
-To deploy the device plugin to your Kubernetes cluster, use the following command:
+----
+## Configuring the AWS Nitro Enclave device plugin for Kubernetes
+
+The device plugin supports the following two configuration options via the device-plugins daemon-set environment
+variables:
+
+### MAX_ENCLAVES_PER_NODE
+Per default AWS EKS nodes can support up to `4` enclaves per node. Number can be reduced if required and evaluated by
+the Kubernetes scheduler.
```

Comment:
I am not quite sure if we should be more definite here in stating that not more than 4 is possible.

Reply:
done